src/sdk/evaluation.ts

```diff
@@ -248,12 +259,24 @@ async function evaluate(
 
     await windupEvaluation(state);
 
-    return {
+    const result: EvaluationResult = {
         run_id: state.eval_run?.runId || '',
         dataset_id: config.dataset_id || state.external_dataset_id,
         session_ids: state.evaluation_session_ids,
-        status: state.state
+        status: state.state,
+        suite: suite,  // Changed from _suite to suite
+        toJson(): void {
```

Comment:
In python, `to_json` generates a file with the inputs/outputs/ground_truths data. In this PR, I though I could match the current return value, and include adding the experiments data in the next PR for supporting passing `ground_truths`.